omc3/model/madx_macros/general.macros.madx

```diff
@@ -71,14 +71,14 @@ do_twiss_monitors(use_sequence, output_file, dpp): macro = {
 * preserve upper case characters.
 * @param dpp: delta p / p to use in the twiss command.
 */
-do_twiss_monitors_and_ips(use_sequence, output_file, dpp): macro = {
+do_twiss_monitors_and_ips(use_sequence, output_file, this_is_the_dpp): macro = {
```

Comment:
Not sure this is better. At least let's update the line above with the new parameter name.

Reply:
Done. I don't like it either, but it could affect anyone that includes `dpp` in their filename path otherwise.

Reply:
I had not thought of that. Good change then, let's update the parameter name and call it a day